pyproject.toml

```diff
@@ -37,6 +37,11 @@ color = true
 preview = true
 
 [tool.cibuildwheel]
+musllinux-aarch64-image = "musllinux_1_2"
```

Comment:
This is optional, but it might be worth adding a comment about why we're using musllinux_1_2.